KA-Flocking/Assets/Scripts/FlockBehaviour.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public abstract class FlockBehaviour : ScriptableObject
-{
-    public abstract Vector3 CalculateMove (FlockAgent agent, List<Transform> context, Flock flock);
-}
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class FlockBehaviour : ScriptableObject
+{
+    public abstract Vector3 CalculateMove (FlockAgent agent, List<Transform> context, Flock flock);
+}
```

Comment:
What happened here? Looks like nothing 🤔

Reply:
Same with all the other `.cs` files. I think you just stole all of @jerge's code ☹️

Reply:
my code now yoink

Reply:
vette fan varför de blir så lul i didnt touch this